pydash/numerical.py

```diff
@@ -383,6 +385,56 @@ def std_deviation(array):
     return math.sqrt(variance(array))
 
 
+def call_math_operator(value1, value2, op):
+    """Return the result of the math operation on the given values."""
+    # Note: If the op is division or multiplication, we need to assign 1 as
+    # default value to the values(needs to be passed in from the outer
+    # methods).
+    try:
+        if not pyd.is_number(value1):
+            value1 = float(value1)
+        if not pyd.is_number(value2):
+            value2 = float(value2)
+    except Exception:
+        pass
+
+    try:
+        return op(value1, value2)
+    except ZeroDivisionError:
+        # The user can pass in the value2 as 0. When the op is division, then
+        # we return INFINITY.
+        return INFINITY
```

Comment:
That's Javascript behavior. Try it with in JS or Node console:

```node
> 1/0
Infinity
```

Python doesn't have this behavior.